nb2/api.py

```diff
@@ -226,6 +230,31 @@ def get(self, slack_user_id, quote_id):
         return marshal(quote, self.fields), 200
 
 
+class PersonQuoteListResource(QuoteResourceBase):
+    """
+    Implements the API method for getting all quotes from a Person.
+
+    The `get` method is implemented to get all Quotes from a Person.
+    """
+
+    ERRORS = {
+        "person_does_not_exist": ("Person with slack_user_id {slack_user_id} " "does not exist"),
```

Comment:
Do we need the multiple quotes here?

Reply:
Oops, I blame `black`.  I probably had it on two lines and it pushed it to one. I'll fix it.